src/main/java/com/temaula/rdb/Periodo.java

```diff
@@ -0,0 +1,64 @@
+package com.temaula.rdb;
+
+import com.temaula.rdb.constraints.CustomConstraint;
+
+import javax.persistence.Embeddable;
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+import javax.validation.constraints.NotNull;
+import java.io.Serializable;
+import java.time.LocalDate;
+import java.util.Objects;
+import java.util.Optional;
+
+@Embeddable
+@CustomConstraint(
+        delegateTo = Periodo.PeriodoValidator.class
+)
+public class Periodo implements Serializable {
```

Comment:
Faz sentido ele ser Serializable?
```suggestion
public class Periodo implements {
```

Reply:
Hehehe, foi força do hábito...heheh vou remover isso